packages/project/src/shared/breakpoints/sort.ts

```diff
@@ -6,6 +6,6 @@ import type { Breakpoints } from "@webstudio-is/project-build";
  */
 export const sort = (breakpoints: Breakpoints) => {
   return Array.from(breakpoints.values()).sort((breakpointA, breakpointB) => {
-    return breakpointA.minWidth - breakpointB.minWidth;
+    return (breakpointA.minWidth ?? 0) - (breakpointB.minWidth ?? 0);
```

Comment:
Still does not consider maxWidth

Reply:
yes, I was thinking about it if a query contains both min and max with, can we really sort when there are intersections?

I think we should not have automated sorting at all and instead allow sorting in the UI like with style sources

Reply:
I don't see a good way to meaningfully sort them automatically when there are combinations of both min and max, I think we should just keep current sorting and later remove it entirely when we have manual sorting ability